rancher/rke2
# Use rancher/wharfie code shared with k3s for for bootstrap

#### Proposed Changes ####

* Remove bootstrap code that has been moved to rancher/wharfie
* Support kubelet image credential plugins (both for bootstrap pull, and for kubelet)
* Use --system-default-registry flag from k3s

Removal of --system-default-registry from agent flags should be called out in release notes, and removed from docs.

#### Types of Changes ####

* Bootstrap
* Private registry

#### Verification ####

* Set --system-default-registry on server, note that agents use same setting
* Configure kubelet image credential plugins, note that bootstrap image and core images can be pulled from ECR if ECR is specified as --system-default-registry

#### Linked Issues ####

For #958 and #959

#### Further Comments ####
